### PR TITLE
New templates for weekly call agendas/notes

### PR DESCRIPTION
**Pull request by flyingzumwalt**
1. New Template for all-hands call
 2. Simplified Template for the Weekly Sprint Issue
 3. Renamed Template for Project-specific sprint calls from `endeavor-etherpad.md` to `agenda-for-project-sprint-calls.md` for clarity & consistency

### CODE REVIEW

File: templates/sprint-issue.md [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-libp2p            | @diasdavid      | 11:30PDT **18:30Z** 20:30CEST | https://public.etherpad-mozilla.org/p/ipfs-__Date__-libp2p
-js-ipfs           | @diasdavid      | 12:00PDT **19:00Z** 21:00CEST | https://public.etherpad-mozilla.org/p/ipfs-__Date__-js-ipfs
-go-ipfs           | @whyrusleeping  | 12:30PDT **19:30Z** 21:30CEST | https://public.etherpad-mozilla.org/p/ipfs-__Date__-go-ipfs
-
-Please add the Agenda to the Pad before the endeavour sprint starts.
-
-## Sprint Deliverables
+All Hands Call   | @RichardLitt    | 9:00PDT **16:00Z** 18:00CEST  | 
+js-ipfs                | @diasdavid       | 10:30PDT **17:30Z** 19:30CEST | https://public.etherpad-mozilla.org/p/ipfs-__Date__-js-ipfs
+go-ipfs               | @whyrusleeping  | 11:00PDT **18:00Z** 20:00CEST | https://public.etherpad-mozilla.org/p/ipfs-__Date__-go-ipfs
```
Comment (jbenet) [inline templates/sprint-issue.md]: we probably need to keep libp2p a dedicated one-- it's pretty big

**Comment by jbenet** [comment]
one comment, otherwie LGTM

**Comment by flyingzumwalt** [comment]
Added libp2p call back into the default schedule.

**Comment by RichardLitt** [comment]
Sweet. Thanks @flyingzumwalt.

**Pull request merged**